### PR TITLE
Hometask HTML02

### PR DESCRIPTION
**Pull request by vladshavel**
later i will try to do additional tasks

### CODE REVIEW

**Comment by vladshavel** [comment]
what does this error mean?
commit can not be built

**Comment by pluhin** [comment]
!!! You have conflicts in your PR, could you please resolve it
(hint: need rebase

**Comment by vladshavel** [comment]
and i have trouble

fatal: Unable to create 'C:/Users/User/Desktop/myworks/sc.it-academy.by/.git/ind ex.lock': File exists.
--

what should i do?

**Comment by vladshavel** [comment]
i think i did everything
but there is one more commit
i made Hometask02html and there are all my task
but then i saw commit "m-sc1-59-19" what is this?>

 MINGW64 ~/Desktop/myworks/sc.it-academy.by (m-sc1-59-19)           
   $ git statu     On branch m-sc1-59-19                                                            
Your branch is ahead of 'origin/m-sc1-59-19' by 2 commits.                         (use "git push" to publish your local commits)                                                                                                                  nothing to commit, working tree clean
--

**Comment by pluhin** [comment]
> i think i did everything
> but there is one more commit
> i made Hometask02html and there are all my task
> but then i saw commit "m-sc1-59-19" what is this?>
> 
> ## MINGW64 ~/Desktop/myworks/sc.it-academy.by (m-sc1-59-19)
> $ git statu On branch m-sc1-59-19
> Your branch is ahead of 'origin/m-sc1-59-19' by 2 commits. (use "git push" to publish your local commits) nothing to commit, working tree clean

It means, you did 2 local commits locally, then did pull - and, i think you got a lots conflicts

**Comment by vladshavel** [comment]
How can I solve this problem?

**Comment by pluhin** [comment]
To solve conflicts you need to do merge locally 
$ git remote add my https://github.com/pluhin/sc.it-academy.by.git
$ git fetch my
$ git merge my/m-sc1-59-19
Here will be many conflicts, solve them, then
$ git add --all 
$ git commit -m "Solve conflicts"
$ git push -f origin m-sc1-59-19

As I see you tried many times, your code is teeming with conflicts

File: vshavel/addpart.html [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+‐‐‐‐‐‐‐‐‐‐‐
+   | х |
+‐‐‐‐‐‐‐‐‐‐‐
+ o | x | o</pre>
+
+ <hr>
+
+<h3>Тег address</h3>
+
+<<<<<<< HEAD
```
Comment (pluhin) [inline vshavel/addpart.html]: Need to remove <<<<<<< HEAD

File: vshavel/addpart.html [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+<<<<<<< HEAD
+<<<<<<< HEAD
+<address>Kuprevicha, 1/1, 220141, Minsk, Belarus. Phone: +375.17.2686911</address>
```
Comment (pluhin) [inline vshavel/addpart.html]: keep just one address

File: vshavel/addpart.html [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+=======
+<address>Kuprevicha, 1/1, 220141, Minsk, Belarus. Phone: +375.17.2686911</address>
+>>>>>>> 5952680... Hometask1
+
+<hr>
+
+<h3>Семантические и функциональные строчные теги</h3>
+
+<p>Пожелания автора:
+<<<<<<< HEAD
```
Comment (pluhin) [inline vshavel/addpart.html]: need to remove <<<<<<< HEAD

File: vshavel/addpart.html [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+<<<<<<< HEAD
+<<<<<<< HEAD
+<br><del style="color:#FF0000">Данный текст убрать из монографии</del> и заменить его на <ins style="color:#0000CD">вновь помещённый текст</ins>.</p>
```
Comment (pluhin) [inline vshavel/addpart.html]: many occurrences of the same code

File: vshavel/addpart.html [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+>>>>>>> ce31093... my tasks
+=======
+<br><del style="color:#FF0000">Данный текст убрать из монографии</del> и заменить его на <ins style="color:#0000CD">вновь помещённый текст</ins>.</p>
+>>>>>>> 5952680... Hometask1
+
+<hr>
+
+<h3>Работа со строчными и блоковыми цитатами</h3>
+<p>Цитата короткая:<br>
+<<<<<<< HEAD
```
Comment (pluhin) [inline vshavel/addpart.html]: need remove this lines

File: vshavel/addpart.html [hunk 6]
```diff
@@ -0,0 +1,224 @@
...
+<<<<<<< HEAD
+<<<<<<< HEAD
+<q><i>И всё­таки она вертится!</i></q></p>
+
+<p>Цитата занимающая несколько предложений:</p>
+
+	<blockquote><p>
+=======
```
Comment (pluhin) [inline vshavel/addpart.html]: need to remove lines like ======

File: vshavel/addpart.html [hunk 7]
```diff
@@ -0,0 +1,224 @@
...
+	<blockquote><p>
+>>>>>>> ce31093... my tasks
+      О сколько нам открытий чудных<br>
+	    Готовят просвещенья дух<br>
+	    И опыт, сын ошибок трудных,<br>
+	    И гений, парадоксов друг,<br>
+<<<<<<< HEAD
+<<<<<<< HEAD
+<<<<<<< HEAD
+	    И случай, бог изобретатель...</p></blockquote>
```
Comment (pluhin) [inline vshavel/addpart.html]: many occurrences of the same code

File: vshavel/addpart.html [hunk 8]
```diff
@@ -0,0 +1,224 @@
...
+>>>>>>> ce31093... my tasks
+
+      <hr>
+
+<h3>Семантические и функциональные строчные теги</h3>
+
+<<<<<<< HEAD
+<<<<<<< HEAD
+<<<<<<< HEAD
+<p><strong>Тэг &#60;STRONG&#62; делает данный фрагмент текста более важным.</strong> <em>Тэг &#60;em&#62; менее важным.</em>
```
Comment (pluhin) [inline vshavel/addpart.html:62]: many occurrences of the same code

File: vshavel/addpart.html [hunk 9]
```diff
@@ -0,0 +1,224 @@
...
+<h3>Теги над­ - и подстрочных символов</h3>
+
+<div align="center">
+
+<<<<<<< HEAD
+<<<<<<< HEAD
+<<<<<<< HEAD
+<header>
+<h3>Магазин работает:</h3></header>
+=======
```
Comment (pluhin) [inline vshavel/addpart.html]: many occurrences of the same code

File: vshavel/addpart.html [hunk 10]
```diff
@@ -0,0 +1,224 @@
...
+
+<hr>
+
+<h3>Тег small внтури абзаца</h3>
+
+<<<<<<< HEAD
+<<<<<<< HEAD
+<<<<<<< HEAD
+<p align="right"><small>Создано при помощи блокнота&#169;</small></p>
+=======
```
Comment (pluhin) [inline vshavel/addpart.html]: many occurrences of the same code

File: vshavel/part1.html [hunk 11]
```diff
@@ -0,0 +1,149 @@
...
+
+<<<<<<< HEAD
+<<<<<<< HEAD
+<<<<<<< HEAD
+<<<<<<< HEAD
+<<<<<<< HEAD
+<<<<<<< HEAD
+<pre><strong><i><u>chmod +x hello.py</u></i></strong></pre>
+=======
+<u><i>chmod +x hello.py</i></u>
```
Comment (pluhin) [inline vshavel/part1.html]: many occurrences of the same code
need to chose one

File: vshavel/part1.html [hunk 12]
```diff
@@ -0,0 +1,149 @@
...
+	<td align="center"><a href="#start"><img src="home.png"></a></td>
+    <td align="right"><a href="part2.html"><img src="nextright.png" align="right"></a></td>
+	
+	</tr>
+
+</table>
+
+
+</body>
+<<<<<<< HEAD
```
Comment (pluhin) [inline vshavel/part1.html]: a lot of not necessary lines

File: vshavel/tables.html [hunk 13]
```diff
@@ -0,0 +1,82 @@
...
+		<th>Minsk</th><td>-5&#176;</td><td>+5<sup><small>o</small></sup></td><td rowspan="2">+2&deg;</td><td>-10&#176;</td>
+	</tr>
+	
+	<tr>
+		<th>Brest</th><td>0&deg;</td><td>+9&#176;</td><td>-4&#176;</td>
+	</tr>
+	
+</table>
+
+<br>
```
Comment (pluhin) [inline vshavel/tables.html:36]: It will better to add hr

**Review by pluhin — CHANGES_REQUESTED** [review]

File: vshavel/tables.html (hunk 13 above)
Comment (vladshavel) [reply vshavel/tables.html:36]: All these heads and others lines
It’s not mine
I will try to solve my problem with git and sent new pull request to you

**Comment by pluhin** [comment]
It was difficult, but i solved all your conflicts.
You can remove your local, and clone again, I push all resolved files to your repo

**Comment by pluhin** [comment]
There is the history:

 git clone git@github.com:vladshavel/sc.it-academy.by.git
plu@plu-laptop MINGW64 /d/work/temp/sc.it-academy.by (m-sc1-59-19)
git remote add my git@github.com:pluhin/sc.it-academy.b
plu@plu-laptop MINGW64 /d/work/temp/sc.it-academy.by (m-sc1-59-19)
$ git fetch my
remote: Enumerating objects: 40, done.
remote: Counting objects: 100% (40/40), done.
remote: Compressing objects: 100% (24/24), done.
remote: Total 45 (delta 16), reused 33 (delta 12), pack-reused 5
Unpacking objects: 100% (45/45), done.
From github.com:pluhin/sc.it-academy.by
 * [new branch]      m-sc1-55-19 -> my/m-sc1-55-19
 * [new branch]      m-sc1-57-18 -> my/m-sc1-57-18
 * [new branch]      m-sc1-58-18 -> my/m-sc1-58-18
 * [new branch]      m-sc1-59-19 -> my/m-sc1-59-19
 * [new branch]      master      -> my/master

plu@plu-laptop MINGW64 /d/work/temp/sc.it-academy.by (m-sc1-59-19)
$ git merge my/m-sc1-59-19
Auto-merging vshavel/part3.html
CONFLICT (add/add): Merge conflict in vshavel/part3.html
Auto-merging vshavel/part2.html
CONFLICT (add/add): Merge conflict in vshavel/part2.html
Auto-merging vshavel/part1.html
CONFLICT (add/add): Merge conflict in vshavel/part1.html
Auto-merging vshavel/addpart.html
CONFLICT (add/add): Merge conflict in vshavel/addpart.html
CONFLICT (modify/delete): dkakhanouski/01_HTML/Kakhanouski_task_01.html deleted in HEAD and modified in my/m-sc1-59-19. Version my/m-sc1-59-19 of dkakhanouski/01_HTML/Kakhanouski_task_01.html left in tree.
CONFLICT (modify/delete): OYarotski/01.HTML/task_02.Python_1.html deleted in HEAD and modified in my/m-sc1-59-19. Version my/m-sc1-59-19 of OYarotski/01.HTML/task_02.Python_1.html left in tree.
Automatic merge failed; fix conflicts and then commit the result.

plu@plu-laptop MINGW64 /d/work/temp/sc.it-academy.by (m-sc1-59-19|MERGING)

plu@plu-laptop MINGW64 /d/work/temp/sc.it-academy.by (m-sc1-59-19|MERGING)
$ git add --all

plu@plu-laptop MINGW64 /d/work/temp/sc.it-academy.by (m-sc1-59-19|MERGING)
$ git commit -m "fix conflicts"
[m-sc1-59-19 b5acc72] fix conflicts

plu@plu-laptop MINGW64 /d/work/temp/sc.it-academy.by (m-sc1-59-19)
$ git push
Enumerating objects: 83, done.
Counting objects: 100% (82/82), done.
Delta compression using up to 4 threads.
Compressing objects: 100% (58/58), done.
Writing objects: 100% (67/67), 27.48 KiB | 1005.00 KiB/s, done.
Total 67 (delta 25), reused 4 (delta 0)
remote: Resolving deltas: 100% (25/25), completed with 7 local objects.
To github.com:vladshavel/sc.it-academy.by.git
   e4d6454..b5acc72  m-sc1-59-19 -> m-sc1-59-19

plu@plu-laptop MINGW64 /d/work/temp/sc.it-academy.by (m-sc1-59-19)

**Comment by vladshavel** [comment]
thank you
can you tell me how can i use git from another PC?
i use PC from my work
i want to use it at home too

**Pull request merged**